File: classesAndObjects.cpp
#include <iostream>
#include <list>
using namespace std;

class YoutubeChannel{
public:
    string Name;
    string OwnerName;
    int SubscribersCount;
    list<string> VideoTitles;

    YoutubeChannel(string name, string onwerName, list<string> titles){
        Name = name;
        OwnerName = onwerName;
        VideoTitles = titles; 
    } 

    void GetInfo(){
        cout<<"__PROPERTIES__"<<endl;
        cout<<" "<<endl;
        cout<<"Channel name: "<< Name <<endl;
        cout<<"The Owner name: "<< OwnerName <<endl;
        for (string i: VideoTitles){
            cout<<"Video title: "<< i <<endl;
        }

    }
};

int main(){
    list<string> videos = {"video 1", "video 2"};

    YoutubeChannel mychannel("MB'Channel", " Masixole Boya", videos);
    mychannel.GetInfo();

    return 0;
}

File: classes_and_objects.py
if __name__ == "__main__":
    
    class Person:
        def __init__(self, name:str, surname:str, age:int)->None:
            self.name = name
            self.surname = surname
            self.age = age

        def introduce_self(self)->None:
            print(f"\nMy name is {self.name}")
            print(f"My surname is {self.surname}")
            print(f"I am {self.age}")
            

    guy1 = Person("Masixole", "Boya", 24)

    guy1.introduce_self()







